### PR TITLE
Abilify Theme Addition

### PR DESCRIPTION
**Pull request by stormedx**
Adding the theme "Abilify" to the repo :^)
here are some direct screenshot examples of the the theme, hope you like!

screenshot-1
screenshot-2

### CODE REVIEW

**Comment by rizwanmustafa** [comment]
Add some more screenshots with different settings on like on #2294

**Comment by stormedx** [comment]
> Add some more screenshots with different settings on like on #2294
sure thing

flipped test colors mode:
flipped-test-colors

Colorful mode:
colorful-mode

**Comment by stormedx** [comment]
oh, also it just occurred to me I don't have an example of typo errors with the theme too, here's that as well
Screenshot 2023-04-23 063901

File: frontend/static/themes/abilify.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    --bg-color: #3b3030;
+    --main-color: #e39d93;
+    --caret-color: #e39d93;
+    --sub-color: #e39d93;
+    --sub-alt-color: #524a4a;
+    --text-color: #e2cebe;
+    --error-color: #d56c6c;
+    --error-extra-color: #ea9999;
+    --colorful-error-color: #cd5b5b;
+    --colorful-error-extra-color: #c27575;
```
Comment (Miodec) [inline frontend/static/themes/abilify.css:11]: Main cannot be the same as sub.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4840793431

**Pull request closed without merging**